GoogleUtilities.podspec

```diff
@@ -4,7 +4,7 @@ Pod::Spec.new do |s|
   s.summary          = 'Google Utilities for iOS (plus community support for macOS and tvOS)'
```

Comment:
GoogleUtilities should get a minor version bump for this change and you should change FirebaseCore to require at least that version.